package.json

```diff
@@ -32,48 +32,51 @@
     "test": "npm run lint-errors && npm run just-test-in-node",
     "test-in-node": "npm run lint-errors && npm run just-test-in-node",
     "just-test": "karma start --config karma.conf.js",
-    "just-test-in-node": "mocha --recursive --compilers js:babel-core/register test/core test/components test/bugs test/swagger-ui-dist-package"
+    "just-test-in-node": "mocha --recursive --compilers js:babel-core/register test/core test/components test/bugs test/swagger-ui-dist-package",
+    "e2e": "nightwatch test/e2e/scenarios --config test/e2e/nightwatch.json --verbose",
+    "e2e-initial-render": "nightwatch test/e2e/scenarios --config test/e2e/nightwatch.json --group initial-render",
+    "mock:api": "json-server --watch test/e2e/db.json --port 3204"
   },
   "dependencies": {
     "base64-js": "^1.2.0",
-    "brace": "0.7.0",
```

Comment:
**Action required:** Please don't bump production dependencies in unrelated PRs 😄